website/search.php

```diff
@@ -44,6 +44,8 @@
 		$bAsKML = (boolean)isset($_GET['polygon_kml']) && $_GET['polygon_kml'];
 		$bAsSVG = (boolean)isset($_GET['polygon_svg']) && $_GET['polygon_svg'];
 		$bAsText = (boolean)isset($_GET['polygon_text']) && $_GET['polygon_text'];
+		$fThreshold = 0.0;
+		if (isset($_GET['polygon_threshold'])) $fThreshold = (float)$_GET['polygon_threshold'];
```

Comment:
It would be nice to have this for format=html, too. Allows to try out the different degrees of simplification directly on the website.